xGame/client/src/Components/MatchSelect/MatchSelect.js

```diff
@@ -1,14 +1,121 @@
 import React from "react";
 import { Collapse } from "reactstrap";
 import PropTypes from "prop-types";
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
```

Comment:
Minor thing, but you can import all of these as
import {Button, TextField, Dialog, DialogContent, DialogContentText, DialogTitle} from '@material-ui/core'